src/lumigo_tracer/lambda_tracer/lambda_reporter.py

```diff
@@ -502,5 +552,32 @@ def _is_edge_kinesis_connection_cache_disabled() -> bool:
     return os.environ.get("LUMIGO_KINESIS_SHOULD_REUSE_CONNECTION", "").lower() == "false"
 
 
+def _should_try_zip() -> bool:
+    return os.environ.get("LUMIGO_SUPPORT_LARGE_INVOCATIONS", "").lower() == "true"
+
+
+def _split_and_zip_spans(spans: List[Dict[Any, Any]]) -> List[str]:
+    """
+    Split spans into bulks and gzip each bulk.
+    """
+    # Start time
+    start_time = time.time()
+    get_logger().debug(f"Splitting the spans into bulks of {MAX_SPANS_BULK_SIZE} spans")
+    spans_bulks = []
+    for i in range(0, len(spans), MAX_SPANS_BULK_SIZE):
+        start_index = i
+        end_index = i + MAX_SPANS_BULK_SIZE
+        bulk = spans[start_index:end_index]
+        zipped_spans = b64encode(gzip_compress(aws_dump(bulk).encode("utf-8"))).decode("utf-8")
+        spans_bulks.append(aws_dump(zipped_spans))
+    # End time and calculate duration
+    end_time = time.time()
+    duration = end_time - start_time
+
+    # Log the execution time
+    get_logger().debug(f"_split_and_zip_spans took {duration:.4f} seconds to execute")
```

Comment:
please change the message to be `Zipping X spans into Y bulks took Z seconds to execute`